3-networks/modules/vpn-ha/main.tf

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  AH VPN configuration
+ *****************************************/
+
+locals {
+  interconnect_project_id = data.google_projects.interconnect_project.projects[0].project_id
+}
+
+data "google_projects" "interconnect_project" {
+  filter = "labels.application_name=prj-interconnect"
+}
+
+module "vpn_ha_region1_router1" {
+  source     = "terraform-google-modules/vpn/google//modules/vpn_ha"
+  project_id = local.interconnect_project_id
```

Comment:
I believe the VPN has to be in the same project as the VPC so this may need to be a parameter.